File: app/src/main/java/bassamalim/hidaya/database/daos/AyatRecitersDao.java
package bassamalim.hidaya.database.daos;

import androidx.room.Dao;
import androidx.room.Query;

import java.util.List;

import bassamalim.hidaya.database.dbs.AyatRecitersDB;

@Dao
public interface AyatRecitersDao {

    @Query("SELECT * FROM ayat_reciters")
    List<AyatRecitersDB> getAll();

    @Query("SELECT rec_name FROM ayat_reciters")
    List<String> getNames();

}

File: app/src/main/java/bassamalim/hidaya/database/daos/AyatRecitersDao.kt
package bassamalim.hidaya.database.daos

import androidx.room.Dao
import androidx.room.Query
import bassamalim.hidaya.database.dbs.AyatRecitersDB

@Dao
interface AyatRecitersDao {
    @Query("SELECT * FROM ayat_reciters")
    fun getAll(): List<AyatRecitersDB>

    @Query("SELECT rec_name FROM ayat_reciters")
    fun getNames(): List<String>
}